src/main/java/de/tum/cit/aet/artemis/communication/web/FaqResource.java

```diff
@@ -184,9 +217,6 @@ public ResponseEntity<Set<FaqDTO>> getFaqForCourse(@PathVariable Long courseId)
     @EnforceAtLeastStudent
     public ResponseEntity<Set<String>> getFaqCategoriesForCourse(@PathVariable Long courseId) {
         log.debug("REST request to get all Faq Categories for the course with id : {}", courseId);
-
-        Course course = courseRepository.findByIdElseThrow(courseId);
```

Comment:
Was this unintentionally there or is there a specific reason this has been removed now?